integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/management/CarbonAlterTableCompactionCommand.scala

```diff
@@ -212,4 +221,56 @@ case class CarbonAlterTableCompactionCommand(
       }
     }
   }
+
+  private def startCompactionForDataMap(carbonLoadModel: CarbonLoadModel,
+      sparkSession: SparkSession): Unit = {
+    val carbonTable = carbonLoadModel.getCarbonDataLoadSchema.getCarbonTable
+    val loadMetaDataDetails = CarbonDataMergerUtil
```

Comment:
correct the coding style in this function, please follow this coding style in future:
```
     val a = foo(
        paramA,
        paramB)
```

Reply:
ok